frontend/src/components/questions-answers/QuestionCard.vue

```diff
@@ -25,11 +25,15 @@ import Card from '@/components/Card'
 export default {
   components: { Card },
   props: {
-    question: Object
+    id: Number,
+    author: Number,
+    likes: Number,
+    title: String,
+    body: String,
+    published: String
```

Comment:
Just to explain this.. when the component is actually rendered the prop has already been transformed to a string in the store. Thus, not a Date obj

Reply:
fair, fixed this in tests now :)